tests/acceptance/EmailManagementCest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+use Page\Acceptance\EmailsPage;
+
+class EmailManagementCest
+{
+    public const ADMIN_PASSWORD   = 'Maut1cR0cks!';
+    public const ADMIN_USER       = 'admin';
+    public const CATEGORY_PREFIX  = 'category #';
+    public const EMAIL_PREFIX     = 'email #';
+    public const EMAILS_TABLE     = 'test_emails';
+    public const CATEGORIES_TABLE = 'test_categories';
+
+    public function _before(AcceptanceTester $I): void
+    {
+        $I->login(self::ADMIN_USER, self::ADMIN_PASSWORD);
+        $this->populateCategoriesTable($I);
+        $this->populateEmailsTable($I);
+    }
+
+    public function tryToBatchChangeEmailCategory(AcceptanceTester $I)
+    {
+        $I->amOnPage(EmailsPage::$URL);
+        $I->waitForElementClickable(EmailsPage::$selectAllCheckbox);
+        $I->click(EmailsPage::$selectAllCheckbox);
+
+        $I->waitForElementClickable(EmailsPage::$selectedActionsDropdown);
+        $I->click(EmailsPage::$selectedActionsDropdown);
+
+        $I->waitForElementClickable(EmailsPage::$changeCategoryAction);
+        $I->click(EmailsPage::$changeCategoryAction);
+
+        $I->waitForElementClickable(EmailsPage::$newCategoryDropdown);
+
+        // Click the dropdown menu
+        $I->click('#email_batch_newCategory_chosen > a > span');
+        $I->waitForElementVisible('#email_batch_newCategory_chosen > div > ul > li.active-result');
```

Comment:
Shouldn't the test assert that the email has the category in the end?

Reply:
Yes... I guess after having so many fights trying to get the click to work I forgot about the reason for the test.

Thanks, I'll fix it